content/ai/concepts/neural-networks/terms/recurrent-neural-networks/recurrent-neural-networks.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'Recurrent Neural Networks'
+Description: 'Recurrent Neural Network is a type of neural network distinguished by its use of the outputs from previous layers as inputs in the current layer'
```

Comment:
Description: 'Recurrent Neural Network is a type of neural network distinguished by storing and re-using outputs from previous layers as additional inputs in the current layer'